File: content/en/api/downtimes/code_snippets/result.api-monitor-schedule-downtime.py

{
    'active': True,
    'canceled': None,
    'creator_id': 3658,
    'disabled': False,
    'end': 1445978817,
    'id': 169267576,
    'message': None,
    'monitor_id': None,
    'monitor_tags': ['*'],
    'parent_id': None,
    'recurrence': {
        'period': 1,
        'type': 'weeks',
        'until_date': 1448387217,
        'until_occurrences': None,
        'week_days': [
            'Mon',
            'Tue',
            'Wed',
            'Thu',
            'Fri'
        ]
    },
    'scope': ['env:staging'],
    'start': 1445968017,
    'updater_id': None
}

# with RRULE recurrence
{
...
'recurrence': {
        'type': 'rrule',
        'rrule': 'FREQ=MONTHLY;BYSETPOS=3;BYDAY=WE;INTERVAL=1'
}

File: content/en/api/downtimes/code_snippets/result.api-monitor-schedule-downtime.rb
[
  '200', {
    'disabled' => false,
    'canceled' => nil,
    'active' => true,
    'message' => nil,
    'id' => 169_267_581,
    'end' => 1_445_978_819,
    'parent_id' => nil,
    'monitor_id' => nil,
    'monitor_tags' => ['*'],
    'recurrence' => {
      'until_date' => 1_448_387_219,
      'until_occurrences' => nil,
      'week_days' => %w[Mon Tue Wed Thu Fri],
      'type' => 'weeks',
      'period' => 1
    },
    'start' => 1_445_968_019,
    'creator_id' => 3658,
    'scope' => ['env:testing'],
    'updater_id' => nil
  }
]

# with RRULE recurrence
{
...
'recurrence': {
    'type' => 'rrule',
    'rrule' => FREQ=MONTHLY;BYSETPOS=3;BYDAY=WE;INTERVAL=1
}
